<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadyInfoDialog</class>
 <widget class="QDialog" name="ReadyInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>PID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="priority">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>优先权级：</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>剩余时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="view_pid">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="view_status">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="view_priority">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="view_time">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="view_memory">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="memory">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>占用主存：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pid_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="pid_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>(KB)</string>
   </property>
  </widget>
  <widget class="QLabel" name="beginAddr">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>主存起址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="view_beginAddr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
